modules/integration_azure-event-hub/README.md

```diff
@@ -95,7 +98,19 @@ Here is the list of required metrics for detectors in this module.
 * `ThrottledRequests`
 
 
+## Notes
 
+### About filtering
```

Comment:
I don't think you should add this note until you change the default filtering.

here is an example of default filtering which does not support the default tagging convention: https://github.com/claranet/terraform-signalfx-detectors/pull/258/files

of course I let you decide what kind of filter you prefer to use to replace missing custom dimensions.